component/src/main/java/com/synopsys/integration/alert/component/authentication/security/AuthenticationHandler.java

```diff
@@ -96,7 +96,7 @@ public SecurityFilterChain filterChain(HttpSecurity http, SecurityContextReposit
             .headers(customizer -> {
                 customizer.contentSecurityPolicy(cspCustomizer -> {
                     cspCustomizer.policyDirectives(
-                        "form-action 'self'; default-src 'none'; connect-src 'self'; object-src 'self'; script-src 'self' 'unsafe-eval'; img-src 'self' data: https://www.synopsys.com/ https://images.synopsys.com/; style-src 'self' 'unsafe-inline'; font-src 'self';"
+                        "form-action 'self'; default-src 'none'; connect-src 'self'; object-src 'self'; script-src 'self' 'unsafe-eval'; img-src 'self' data: https://www.synopsys.com/ https://images.synopsys.com/ https://www.blackduck.com/; style-src 'self' 'unsafe-inline'; font-src 'self';"
```

Comment:
Should these instances of Synopsys be removed or is this part of the spike, IALERT-3704?

Reply:
Should be safe to remove as part of 8.0.0

Reply:
No This is part of IALERT-3760.  That is specific to updating the CSP header to remove the references to Synopsys.